src/preprocessor.ts

```diff
@@ -30,6 +27,7 @@ export function process(
     const isTsFile = path.endsWith('.ts') || path.endsWith('.tsx');
     const isJsFile = path.endsWith('.js') || path.endsWith('.jsx');
     const isHtmlFile = path.endsWith('.html');
+	postHook = getPostProcessHook(compilerOptions, config, tsJestConfig);
```

Comment:
minor nit - indentation.

should there be a `if postHook !==  undefined` around this? If not, is there any reason to declare it outside this function?